### DISCUSSION TITLE
新构建中，为windows平台提供ipv6支持等建议

### DISCUSSION THREAD

**Discussion by huayuprophet** [opengnb/opengnb#32] [Ideas]
未来版本希望支持ipv6，并且为了适应更加复杂的网络路由环境，建议p2p组网时使用udp协议或者基于udp协议封装的quic协议实现打通和通讯。

**Comment by gnbdev** [comment]
第一点：
opengnb 的wintun模块时可以支持ipv6
你可以关注 模块开发者的仓库 https://www.github.com/wuqiong

第二点：
请阅读源码

第三点：
如果需要quic，那更合适的做法是设计一个私有协议。
